tox/config.py

```diff
@@ -787,7 +787,8 @@ def __init__(self, config, inipath):
             factors = set(name.split('-'))
             if section in self._cfg or factors <= known_factors:
                 config.envconfigs[name] = \
-                    self.make_envconfig(name, section, reader._subs, config)
+                    self.make_envconfig(name, section, reader._subs, config,
+                                        replace=name in config.envlist)
```

Comment:
Actually, my intent is to use `replace=False` when the environment is not scheduled in the current run.  I'm not 100% sure `config.envlist` is what I want here.